packages/realm-react/src/__tests__/helpers.ts

```diff
@@ -0,0 +1,75 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2020 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+import { AppConfig, AppImporter, Credentials } from "@realm/app-importer";
+import { act, waitFor } from "@testing-library/react-native";
+import { useAuth } from "../useAuth";
+import { useEmailPasswordAuth } from "../useEmailPasswordAuth";
+
+export async function testAuthOperation({
+  authOperation,
+  result,
+  expectedResult,
+}: {
+  authOperation: () => Promise<any>;
+  result: { current: ReturnType<typeof useEmailPasswordAuth> | ReturnType<typeof useAuth> };
+  expectedResult: () => void;
+}) {
+  await act(async () => {
+    authOperation();
+    await waitFor(() => {
+      expect(result.current.result.pending).toEqual(true);
+    });
+  });
+  await waitFor(() => {
+    expectedResult();
+  });
+}
+
+const { realmBaseUrl = "http://localhost:9090" } = process.env;
+
+export const baseUrl = realmBaseUrl;
+
+function getCredentials(): Credentials {
+  const { publicKey, privateKey, username = "unique_user@domain.com", password = "password" } = process.env;
```

Comment:
Still feel these names are a bit too general for the `process.env` namespace, would like an `baas` prefix or similar .. also environment variables tend to be "ALL_CAPS".

Reply:
We should address this globally then, as this is not unique to here

Reply:
This is the first place they're read from `process.env` .. in the integration tests they are read from `environment` (I know - not the best name, but `context` was sort of taken by Mocha) which is passed through the `MOCHA_REMOTE_CONTEXT` environment variable.

Reply:
> I could also read them out of `CONTEXT`

That would make the tests align more with the integration tests 👍